app/src/test/java/com/globant/onboardingcalculatorwithkotlin/CalculatorPresenterTest.kt

```diff
@@ -382,6 +382,61 @@ class CalculatorPresenterTest {
         assertEquals(EMPTY_CHAR, model.operator)
         assertEquals(EMPTY_STRING, model.second_operand)
         assertEquals(EMPTY_STRING, model.result)
+    }
+
+    @Test
+    fun `on delete button pressed with empty operation show error message`(){
+        presenter.onDeletePressed()
+
+        assertEquals(EMPTY_STRING, model.first_operand)
+        assertEquals(EMPTY_CHAR, model.operator)
+        assertEquals(EMPTY_STRING, model.second_operand)
+        assertEquals(EMPTY_STRING, model.result)
+
+        verify(mockedView).showMessage(CalculatorError.INCOMPLETE_OPERATION)
+    }
+
+    @Test
+    fun `on delete button pressed with first operand`(){
+        model.first_operand = "$NUMBER_FIVE$NUMBER_EIGHT$NUMBER_ZERO"
+
+        presenter.onDeletePressed()
+
+        assertEquals("$NUMBER_FIVE$NUMBER_EIGHT", model.first_operand)
```

Comment:
is it useful to do an assert on a value assigned two lines before?

Reply:
Changed!